plugin/ctrlsf.vim

```diff
@@ -134,6 +134,7 @@ let s:default_mapping = {
     \ "next"  : "<C-J>",
     \ "prev"  : "<C-K>",
     \ "pquit" : "q",
+    \ "llist" : "Q",
```

Comment:
Due to the nature of this map does little things, and there is a chance that user has already defined a global mapping for `lopen`, I think it's better to leave this default map empty.